docs/contributing.md

```diff
@@ -163,26 +163,24 @@ To use a preview build for a package within a project, you need to override the
 
 The [`create-release-branch`](https://github.com/MetaMask/create-release-branch) tool and [`action-publish-release`](https://github.com/MetaMask/action-publish-release) GitHub action are used to automate the release process.
 
-1. **Create a release branch.**
+1. **Initiate the Release Branch and Pre-Update Changelogs**
 
-   Run `yarn create-release-branch`. This tool generates a file and opens it in your editor, where you can specify which packages you want to include in the next release and which versions they should receive. Instructions are provided for you at the top; read them and update the file accordingly.
+   Run `yarn create-release-branch`. This command creates a release branch named `release/<new release version>` and pre-updates the changelogs for all packages with changes. It also generates a file and opens it in your editor, allowing you to specify which packages you want to include in the next release and their versions. Follow the instructions at the top of the file to update it accordingly.
 
-   When you're ready to continue, save and close the file.
+2. **Review and update changelogs for relevant packages.**
 
-2. **Update changelogs for relevant packages.**
+   First, review all pre-updated changelogs to understand the recent changes. Then, for the packages you intend to release, update these sections in each changelog:
```

Comment:
Once again, we can drop the "pre-" here. I also think we can smooth the transition between this step and previous step:

``` suggestion
   At this point, the versions of all packages you intend to release have been bumped and their changelogs list new changes. Now you need to go through each changelog and make sure that they follow existing standards: